### DISCUSSION TITLE
Slow to start reviewing flashcards.

### DISCUSSION THREAD

**Discussion by yasuhito** [st3v3nmw/obsidian-spaced-repetition#662] [Q&A]
When reviewing flashcards, the deck selection modal appears immediately, but it takes time for the review to begin after clicking on a deck. In my experience, the first flash card appears after 20-30 seconds after several successive clicks. Once the review begins, there are no performance issues after that. Is anyone else having the same problem? By the way, I currently have about 200 flashcards, which I don't consider a very large number.

**Comment by Newdea** [comment]
maybe you have too many files in your vault
